contrib/scoverage/src/mill/contrib/scoverage/ScoverageModule.scala

```diff
@@ -211,9 +205,9 @@ trait ScoverageModule extends ScalaModule { outer: ScalaModule =>
       T {
         val extras =
           if (isScala3()) {
-            Seq(s"-coverage-out:${data().path.toIO.getPath()}")
+            Seq(s"-coverage-out:${data().path.toIO.getPath}")
```

Comment:
I'd prefer to call methods as they are defined, and Java API methods always have exactly one parameter list. I don't understand why correctness is treated here. In Scala, methods can have none or multiple parameter lists, so we should always be extra careful to use the exact version.

Reply:
I don't know but IDEA complains about it... I can rollback it